az-digital/az_quickstart
# Closes #2855: Full-width background padding

## Description
Looking to align content within a full width background to other content on the page. Add demo content to cover these testing needs.

## Related issues
Closes #2855 

## How to test

1. Add full-width background to pages with and without sidebar nav
   a. `/pages/combo-page`
   b. `/pages/combo-page-no-sidebar`
4. Verify main content and full-width content align at widths

https://094aab32-6c3a-47f7-8953-f5415d7033d0.probo.build/

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
